### ISSUE TITLE
Cannot use JSON_VALUE function

### ISSUE THREAD

**Issue by arnaudbriche** [chdb-io/chdb#270]
(you don't have to strictly follow this form)

**Describe the unexpected behaviour**
A clear and concise description of what works not as it is supposed to.

**How to reproduce**

```sql
SELECT JSON_VALUE('{"key": "value"}', '$.key')
``` 

* Which ClickHouse server version to use
* Which interface to use, if matters

chdb

* Non-default settings, if any
* `CREATE TABLE` statements for all tables involved
* Sample data for all these tables, use [clickhouse-obfuscator](https://github.com/ClickHouse/ClickHouse/blob/master/programs/obfuscator/Obfuscator.cpp#L42-L80) if necessary
* Queries to run that lead to unexpected result

**Expected behavior**

**Error message and/or stacktrace**

```
chdb.ChdbError: Code: 48. DB::Exception: Functions JSON* are not supported: In scope SELECT JSONExtract('{"key": "value"}', 'Tuple(String)'), JSON_VALUE('{"key": "value"}', '$.key'). (NOT_IMPLEMENTED)
```

**Additional context**

**Comment by auxten** [comment]
It a macOS compiling flags issue, should be issue of [build.sh](https://github.com/chdb-io/chdb/blob/main/chdb/build.sh)

**Comment by auxten** [comment]
Fixed #271
